src/common.jl

```diff
@@ -80,6 +74,33 @@ function solve{uType,tType,isinplace,algType<:ODEJLAlgorithm,F}(prob::AbstractOD
                           minstep=dtmin,
                           initstep=dt,
                           points=points)
+    elseif typeof(alg) <: ode4
+        ts,timeseries_tmp = ODE.ode4(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
+    elseif typeof(alg) <: ode4ms
+        ts,timeseries_tmp = ODE.ode4ms(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
+    elseif typeof(alg) <: ode4s
+        ts,timeseries_tmp = ODE.ode4s(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
     end
```

Comment:
I'm curious why removing the type declaration would be a problem for usage with Sundials.jl?  Even if this function doesn't have a type on the parameter, those solvers could be passed in.  Or are you trying to prevent, e.g., a DAE algorithm from being passed into a ODE solver?

Either way, if there are only a few `ode*` algorithms which don't accept `kwargs`, then yes, I think giving all of them `kwargs` (and ignoring thes parameters if necessary) is worthwhile, since it let's you simplify this code considerably.